packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -1,64 +1,165 @@
+                  label="Issues"
+                  expression='@jisourcetype AND NOT @jidocumenttype="WorkLog"'
+                  engine-id={engineId}
+                ></c-quantic-tab>
+                <c-quantic-tab
+                  label="Community"
+                  expression='@objecttype=="Message"'
+                  engine-id={engineId}
+                ></c-quantic-tab>
+                <c-quantic-tab
+                  label="Files"
+                  expression="@boxdocumenttype==File OR @spcontenttype==Document"
+                  engine-id={engineId}
+                ></c-quantic-tab>
               </c-quantic-tab-bar>
               <div class="slds-grid slds-var-m-top_small">
-                <c-quantic-summary class="slds-var-m-vertical_x-small slds-small-size_8-of-12 slds-large-size_7-of-12" engine-id={engineId}>
+                <c-quantic-summary
+                  class="slds-var-m-vertical_x-small slds-small-size_8-of-12 slds-large-size_7-of-12"
+                  engine-id={engineId}
+                >
                 </c-quantic-summary>
-                <c-quantic-sort class="slds-var-m-vertical_xxx-small slds-large-size_5-of-12 slds-show_large" engine-id={engineId}>
+                <c-quantic-sort
+                  class="slds-var-m-vertical_xxx-small slds-large-size_5-of-12 slds-show_large"
+                  engine-id={engineId}
+                >
                 </c-quantic-sort>
-                <div class="slds-small-size_4-of-12 slds-clearfix slds-hide_large">
-                    <div class="slds-float_right">
-                      <c-quantic-refine-toggle engine-id={engineId} full-screen></c-quantic-refine-toggle>
-                    </div>
+                <div
+                  class="slds-small-size_4-of-12 slds-clearfix slds-hide_large"
+                >
+                  <div class="slds-float_right">
+                    <c-quantic-refine-toggle
+                      engine-id={engineId}
+                      full-screen
+                    ></c-quantic-refine-toggle>
+                  </div>
                 </div>
               </div>
-              <c-quantic-query-error engine-id={engineId}></c-quantic-query-error>
-              <c-quantic-breadcrumb-manager engine-id={engineId}></c-quantic-breadcrumb-manager>
-              <c-quantic-did-you-mean engine-id={engineId}></c-quantic-did-you-mean>
+              <c-quantic-query-error
+                engine-id={engineId}
+              ></c-quantic-query-error>
+              <c-quantic-breadcrumb-manager
+                engine-id={engineId}
+              ></c-quantic-breadcrumb-manager>
+              <c-quantic-did-you-mean
+                engine-id={engineId}
+              ></c-quantic-did-you-mean>
               <c-quantic-no-results engine-id={engineId}></c-quantic-no-results>
-              <c-quantic-result-list engine-id={engineId}></c-quantic-result-list>
+              <div class="slds-var-m-vertical_small">
```

Comment:
Why the margin is only around the QuanticSmartSnippet component? it's possible that a query returns some  `smart snippet suggestions` but no `smart snippet`, in that case maybe we need margin around the QuanticSmartSnippetSuggestions as well.
It would be nice to see this situation as well in a screenshot.